gui/src/tutorialoverlay.cpp

```diff
@@ -181,21 +175,13 @@ void TutorialOverlay::buildUi()
 	qDebug(CAT_TUTORIALOVERLAY)<<"build";
 	overlay = new TintedOverlay(parent);
 
-	ui = new Ui::Tutorial();
-	ui->setupUi(this);
-	ui->description->setStyleSheet("background-color: rgba(0,0,0,60);color: white;");
-	ui->title->setStyleSheet("color:white");
-	ui->btnExit->setStyleSheet("width:80;height:20");
-	ui->btnExit->setProperty("blue_button", true);
-	ui->btnNext->setStyleSheet("width:80;height:20");
-	ui->btnNext->setProperty("blue_button", true);
-	ui->btnNext->setFocus();
-	connect(ui->btnNext,&QPushButton::clicked,this,&TutorialOverlay::next);
-	connect(ui->btnExit, &QPushButton::clicked, this, &TutorialOverlay::abort);
+	m_popupWidget = new PopupWidget();
```

Comment:
Are you reusing the same popup widget for each of the tutorial steps ? 
Wouldn't it be better if each step had it's own popup ? I think it already has it's own hoverwidget ..